### DISCUSSION TITLE
Two dates in one html form

### DISCUSSION THREAD

**Discussion by jonogr** [mohamadadithya/calendarify#6] [Q&A]
Hi, I really like this calendar, nice work.  Does it support two dates in one html form?  Something like below, but this example only seems to enable the calendar on the enddate.

<form>
    <input id="startdate" class="startdate" name="startdate" type="text">
    <input id="enddate" class="enddate" name="enddate" type="text">
</form>
<script>
  const startdate = new Calendarify('.startdate', {quickActions: false})
  startdate.init()
  const enddate = new Calendarify('.enddate', {quickActions: false})
  enddate.init()
</script>

Many thanks.

**Comment by mohamadadithya** [comment]
I'll working on that, thanks for asking @jonogr .

**Reply by jonogr** [reply]
Thanks! That would be really useful.

**Comment by jonogr** [comment]
Hi there, I still like using this calendar, but any further thoughts on how I could use it twice in the same form?
